example/demo.js

```diff
@@ -1,9 +1,9 @@
-const hacktoberfest = require('../lib/index'),
+const hacktoberfest = require("../lib/index"),
   options = {
-    token: 'XXXXXXXXXXXXXXX', // replace it with your token
-    labels: 'first-timers-only',
+    token: "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", // replace it with your token
```

Comment:
@whddnjs1715 Please do not share the token online.

Reply:
Thank you very much 
i appreciate that